semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -181,12 +221,13 @@ always equal to an asterisk (`*`).
 
 ```protobuf
 message SymbolInformation {
+  reserved 6;
   string symbol = 1;
   string language = 2;
   Kind kind = 3;
   int32 properties = 4;
   string name = 5;
-  Range range = 6;
+  Location location = 10;
```

Comment:
Instead of deprecating `range`, I removed it. This is an exceptional case, but that field never worked properly, and it was introduced just a couple days ago, so I think that it's alright to outright remove it.

Reply:
👍